<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GrDiv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Div</string>
    </property>
    <widget class="QLineEdit" name="TFDiv1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Div3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div7</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Div8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Div8</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFDiv8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GrMod">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Mod</string>
    </property>
    <widget class="QLineEdit" name="TFMod1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Mod3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod6</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod7</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Mod8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mod8</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFMod8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GrFunct">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Function</string>
    </property>
    <widget class="QLabel" name="InpPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Path</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TFInpPath">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BtInp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderNew"/>
     </property>
    </widget>
    <widget class="QPushButton" name="Encryption">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Encryption</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Decryption">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Decryption</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
